api/envoy/api/v2/route/route_components.proto

```diff
@@ -143,8 +143,20 @@ message VirtualHost {
   // This header is unaffected by the
   // :ref:`suppress_envoy_headers
   // <envoy_api_field_config.filter.http.router.v2.Router.suppress_envoy_headers>` flag.
+  //
+  // [#next-major-version: rename to include_attempt_count_in_request.]
```

Comment:
You can do this now with `[(udpa.annotations.field_migrate).rename` right? Or is the issue that this will right now change it in v3 and we have to circle back and do it after we stop v2 updates?

Reply:
there's a wrinkle with the annotation right now https://github.com/envoyproxy/envoy/pull/10325#discussion_r393212597. I am keeping a note so that when this is fix we can change to use the annotation.